docs/layouts/shortcodes/generated/rest_v1_dispatcher.html

```diff
@@ -101,10 +101,10 @@
       "type" : "object",
       "id" : "urn:jsonschema:org:apache:flink:runtime:rest:messages:DashboardConfiguration:Features",
       "properties" : {
-        "web-submit" : {
```

Comment:
Out of curiosity. Do you know where does the reordering comes from? Is the file autogenerated? If you don't know don't bother looking for an answer.

Reply:
This file is autogenerated but it looks like it is generated in alphabet order so it should be correct now. I can only guess but perhaps, initially, the author made these changes manually